src/mococrw/key.h

```diff
@@ -43,9 +43,9 @@ class AsymmetricKey
     /**
      * Supported asymmetric key types.
      */
-    enum class KeyTypes : int { RSA = EVP_PKEY_RSA, ECC = EVP_PKEY_EC };
+    enum class KeyTypes : int { RSA, ECC, UNSUPPORTED };
```

Comment:
We could create a key type ECC_ED for the new curves? I am not sure about pro and con. 
For better ABI/API stability, UNSUPPORTED should get a high enum value?